CSC8503/GameTechRenderer.cpp

```diff
@@ -535,16 +535,21 @@ void GameTechRenderer::DrawOutlinedObjects() {
 	glUniform2f(glGetUniformLocation(mOutlineShader->GetProgramID(), "pixelSize"), 1.0f / hostWindow.GetScreenSize().x, 1.0f / hostWindow.GetScreenSize().y);
 
 	for (int i = 0; i < mOutlinedObjects.size(); i++) {
+
+		glUniform1i(glGetUniformLocation(mOutlineShader->GetProgramID(), "hasAnim"), mOutlinedObjects[i]->GetAnimation() ? 1 : 0);
```

Comment:
How about seperating parameters like this:

auto& location = glGetUniformLocation(mOutlineShader->GetProgramID(), "hasAnim");
bool hasAnimation = mOutlinedObjects[i]->GetAnimation() ? 1 : 0;
glUniform1i(location ,hasAnimation  );